ui/static/ui/js/listing.js

```diff
@@ -54,5 +126,96 @@ define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
         event.preventDefault();
       }
     });
+
+    //open the lateral panel for members
+    $('.btn-members').on('click', function(event) {
+      event.preventDefault();
+      //remove any alert
+      $('#members-alert').empty();
+      //reset the form values
+      resetUserGroupForm();
+      //make panel visible
+      $('.cd-panel-members').addClass('is-visible');
+      //retrieve all the members
+      showUpdateAllMembers();
+      //
+    });
+    //close the lateral panel for members
+    $('.cd-panel-members').on('click', function(event) {
+      if ($(event.target).is('.cd-panel-members') ||
+          $(event.target).is('.cd-panel-close')) {
+        $('.cd-panel-members').removeClass('is-visible');
+        event.preventDefault();
+      }
+    });
+    //add button for the members
+    $(document).on('click', '.cd-panel-members-add', function() {
```

Comment:
because the HTML is injected into the page dynamically and unless you attach the event handler to the document, it will not work